src/ZenithAngleCalc.jl

```diff
@@ -204,103 +28,88 @@ function instantaneous_zenith_angle(date::DateTime,
     ϕ = deg2rad(latitude)
     γ = deg2rad(obliquity)
     ϖ = deg2rad(perihelion)
-    ecc = eccentricity
+    e = eccentricity
 
-    TA = true_anomaly(date)
+    # mean anomaly at vernal equinox, radians (3.11)
+    β = (1-e^2)^(0.5)
+    MAve = -ϖ + (e + e^3/4)*(1+β)*sin(ϖ) - 0.5*e^2*(0.5+β)*sin(2*ϖ) + 0.25*e^3*(1/3+β)*sin(3*ϖ)
+
+    # mean anomaly, radians (3.10)
+    julian_day_abs = datetime2julian(date)
+    julian_day_ref = 2451545.0 # Jan 1, 2000 at 12hr
+    julian_day_ve = julian_day_ref + 79.0 # vernal equinox 2000
+    t_since_vernal_equinox = mod(julian_day_abs - julian_day_ve, year_anom())
+    MA = 2π * t_since_vernal_equinox / year_anom() + MAve
+
+    # true anomaly, radians (3.8)
+    TA = MA + (2*e - e^3/4)*sin(MA) + (5/4)*e^2*sin(2*MA) + (13/12)*e^3*sin(3*MA)
+
+    # true longitude, radians (3.9)
     TL = mod(TA + ϖ, 2*π)
-    GMSTrad = GMST(date, timezone)
-    d = earth_sun_dist(ecc, TA)
 
-    # declination, radians
+    # declination, radians (3.16)
     δ = mod(asin(sin(γ) * sin(TL)), 2*π)
-    # right acension, radians
-    RA = mod(atan(cos(γ) * sin(TL) / cos(TL)), 2*π)
 
     # hour angle, radians
```

Comment:
```suggestion
    # hour angle, radians (3.17)
```
(It's not exactly the same but I think it's still helpful to add the equation number.)